sources/packages/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.controller.ts

```diff
@@ -51,21 +54,29 @@ export class ConfirmationOfEnrollmentController {
    * @returns student application list of an institution location
    */
   @HasLocationAccess("locationId")
-  @Get(":locationId/confirmation-of-enrollment")
+  @Get(
+    ":locationId/confirmation-of-enrollment/enrollmentPeriod/:enrollmentPeriod",
+  )
   async getCOESummary(
     @Param("locationId") locationId: number,
+    @Param("enrollmentPeriod") enrollmentPeriod: string,
   ): Promise<COESummaryDTO[]> {
-    const applications = await this.applicationService.getCOEApplications(
-      locationId,
-    );
-    return applications.map((eachApplication: Application) => {
+    const disbursementSchedules =
+      await this.disbursementScheduleService.getCOEByLocation(
+        locationId,
+        enrollmentPeriod === EnrollmentPeriod.Upcoming,
```

Comment:
If there is an enum to define the period, please consider using the same enum as a parameter for the method getCOEByLocation.

Reply:
Addressed.